dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/trackedentity/DefaultTrackedEntityInstanceService.java

```diff
@@ -135,7 +135,11 @@
         }
 
         decideAccess( params );
-        validate( params );
+        //AccessValidation should be skipped only and only if it is internal service that runs the task (for example sync job)
+        if ( !skipAccessValidation )

```

Comment:
I am not sure if we should skip access validation. Could it be better to ensure that the user used for import during sync simply has the required auths?